test/sdk/restapi/http-api-test.go.nolint

```diff
@@ -0,0 +1,96 @@
+// Copyright 2019 Google LLC All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	"agones.dev/agones/test/sdk/restapi/swagger"
+	"golang.org/x/net/context"
+)
+
+func main() {
+	log.Println("Client is starting")
+	time.Sleep(1 * time.Second)
+	conf := swagger.NewConfiguration()
+	portStr := os.Getenv("AGONES_SDK_HTTP_PORT")
+	conf.BasePath = "http://localhost:" + portStr
+	cli := swagger.NewAPIClient(conf)
+
+	ctx := context.Background()
+	uid := ""
+
+	// Wait for SDK server to start (5 seconds)
+	for c := 0 ; c < 5; c++ {
+		_, _, err := cli.SDKApi.Ready(ctx, swagger.SdkEmpty{})
+		if err == nil {
+			break
+		} else {
+			log.Printf("Could not send Ready: %v\n", err)
+		}
+		time.Sleep(1 * time.Second)
+	}
+
+	go func() {
+		gs, _, err := cli.SDKApi.WatchGameServer(ctx)
+		if err != nil {
+			log.Fatalf("Error in WatchGameServer: %v\n", err)
```

Comment:
I think log.Fatal should only happen in the main goroutine. Can this pass the error back to the main goroutine instead of a uid to signal that it failed?